commons/protos/policy_interface.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+message Node {
+  string version = 1;
+  string image = 2;
+  map<string, string> metadata = 3;
+}
+
+message Edge {
+  uint64 from = 1;
+  uint64 to = 2;
+}
+
+message Graph {
+  repeated Node nodes = 1;
+  repeated Edge edges = 2;
+}
+
+message PluginInput {
+  Graph graph = 1;
+  map<string, string> parameters = 2;
+}
```

Comment:
Missing:
* PluginOutput containing:  with outgoing parameters and a PluginError
* PluginError containing a number and a string